rubrics/impl/src/main/java/org/sakaiproject/rubrics/impl/repository/EvaluationRepositoryImpl.java

```diff
@@ -29,17 +29,23 @@
 import org.sakaiproject.rubrics.api.model.ToolItemRubricAssociation;
 import org.sakaiproject.rubrics.api.repository.EvaluationRepository;
 import org.sakaiproject.springframework.data.SpringCrudRepositoryImpl;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
```

Comment:
Don't add logging like this. Take a look at the rubricsservice, how I use the lombok.slf4j.Slf4j annotation. That annotation adds the log property to your class with nothing else needing to be done.